packages/grafana-data/src/types/fieldOverrides.ts

```diff
@@ -50,10 +50,10 @@ export interface FieldConfigEditorConfig<TOptions, TSettings = any, TValue = any
   shouldApply?: (field: Field) => boolean;
 
   /** Indicates that option shoukd not be available in the Field config tab */
-  hideFromFieldOptions?: boolean;
+  hideFromDefaults?: boolean;
```

Comment:
nit: I'm probably missing some context or a naming convention here but I find the previous name easier to follow.

Reply:
@jackw to shed some light on this. When we talk about "default field options" we refer to the options that are available from the Fields tab in the panel editor's side pane. This is pretty much a naming convention that we use, hence this suggestion from Torkel.